hardinfo/info.c

```diff
@@ -56,6 +56,33 @@ void info_add_group(struct Info *info, const gchar *group_name, ...)
     g_array_append_val(info->groups, group);
 }
 
+void info_add_group_ff(struct Info *info, const gchar *group_name, ...)
```

Comment:
What's `ff` supposed to mean?

Reply:
Fast forward. I didn't have a name mind. This pr isn't a real one, it was mostly just for discussion.

Reply:
See #180